scipy/ndimage/src/_rank_filter_1d.cpp

```diff
@@ -315,19 +319,17 @@ static struct PyModuleDef _rank_filter_1d = {
     myMethods};
 
 // init the module
-PyMODINIT_FUNC
-PyInit__rank_filter_1d(void)
-{
-    PyObject *module;
+PyMODINIT_FUNC PyInit__rank_filter_1d(void) {
```

Comment:
Please try not to mix in style changes with bug fixes in the same commit. This change actually makes the code less idiomatic (see, e.g., https://docs.python.org/3/c-api/intro.html#c.PyMODINIT_FUNC) so please revert this one.

Indentation changes to match the style in a single file are okay, but best done in a separate PR or at least a separate commit.